pandas/indexes/multi.py

```diff
@@ -864,9 +862,9 @@ def from_arrays(cls, arrays, sortorder=None, names=None):
             if len(arrays[i]) != len(arrays[i - 1]):
                 raise ValueError('all arrays must be same length')
 
-        cats = [Categorical.from_array(arr, ordered=True) for arr in arrays]
```

Comment:
ideally we *should* replace these and deprecate the constructor. but can do that later / another PR if desired.